src/main/java/domain/Number.java

```diff
@@ -0,0 +1,17 @@
+package domain;
+
+public record Number(int value) {
+
+    public static final int MIN_NUMBER = 1;
+    public static final int MAX_NUMBER = 45;
```

Comment:
```suggestion
    private static final int MIN_NUMBER = 1;
    private static final int MAX_NUMBER = 45;
```
외부에서 사용되지 않는 상수는 private으로 막아주는 것이 `Number`라는 도메인만이 가지는 온전한 정보가 될 수 있습니다.

테스트코드에 활용할때에는 이 값을 활용하는 것이 아니라 순수한 값을 사용하여 검증하면 되지 않을까요? `MIN_NUMBER`가 실수로 2로 바뀌었는데 이 상수를 테스트에서 그대로 사용하고 있었다면 잘못된 코드를 확인하기 어렵지 않았을까요?

Reply:
단순히 상수 값이 바뀔 경우에 테스트 코드를 바꾸지 않아도 된다는 생각에 public으로 작성했던 것 같습니다. 말씀해주신 것처럼 테스트의 목적을 생각했을 때, 순수한 값을 사용하는 것이 좋다고 느껴집니다. 변경하겠습니다.